components/bt/bluedroid/btc/profile/std/avrc/btc_avrc.c

```diff
@@ -250,6 +252,77 @@ static void handle_rc_disconnect (tBTA_AV_RC_CLOSE *p_rc_close)
 #endif
 }
 
+static void handle_rc_attributes_rsp ( tAVRC_MSG_VENDOR* vendor_msg){
+    uint8_t attr_count = vendor_msg->p_vendor_data[4];
+    int attr_index = 5;
+    int attr_length = 0;
+    uint32_t attr_id = 0;
+
+    //Check if there are any attributes
+    if(attr_count < 1) return;
+
+    esp_avrc_ct_cb_param_t param[attr_count];
+    memset(&param[0], 0, sizeof(esp_avrc_ct_cb_param_t) * attr_count);
+
+  for(int i = 0; i < attr_count; i++){
+    attr_length = (int) vendor_msg->p_vendor_data[7 + attr_index] | vendor_msg->p_vendor_data[6 + attr_index] << 8;
+
+    //Received attribute text is not null terminated, so it's useful to know it's length
+    param[i].meta_rsp.attr_length = attr_length;
+    param[i].meta_rsp.attr_text = &vendor_msg->p_vendor_data[8 + attr_index];
+
```

Comment:
The code indentation need to be fixed. You can use the script $IDF_PATH/tools/format.sh to fix the typos.

Reply:
I ran the script on all the files I modified in this PR, should be ok now.